crates/language/src/language_settings.rs

```diff
@@ -4,6 +4,7 @@ use crate::{File, Language, LanguageServerName};
 use anyhow::Result;
 use collections::{HashMap, HashSet};
 use core::slice;
+use ec4rs::property::{FinalNewline, IndentSize, IndentStyle, TabWidth, TrimTrailingWs};
```

Comment:
`MaxLineLen` seems like `preferred_line_length` and `soft_wrap` combination, doesn't it?
https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#max_line_length

So let's support it too.

Reply:
That was not in the spec, well caught!